_includes/downloads/opensearch-docker.markdown

```diff
@@ -1,7 +1,8 @@
 The best way to try out OpenSearch is to use [Docker Compose](https://docs.docker.com/compose/install/). These steps will setup a two node cluster of OpenSearch plus OpenSearch Dashboards:
 
 1. Download [docker-compose.yml](https://opensearch.org/samples/docker-compose.yml) into your desired directory
-2. Run `docker-compose up`
-3. Have a nice coffee while everything is downloading and starting up
-4. Navigate to [http://localhost:5601/](http://localhost:5601) for OpenSearch Dashboards
-5. Login with the default username (`admin`) and password (`admin`)
+2. Make sure you have Docker set to use at least 4 GB of RAM in **Preferences** > **Resources**.
```

Comment:
Is this Windows only? Desktop version? How do I do this on Linux command line?

Reply:
This is exactly how it's stated in documentation - if we want to change this line we should go docs first.

Reply:
The above step is only for Windows/OSX user. @stockholmux you missed for Linux/Debian users as mentioned here https://github.com/opensearch-project/OpenSearch/issues/978#issuecomment-885228338

Reply:
@owaiskazi19 See https://github.com/opensearch-project/OpenSearch/issues/978#issuecomment-885252386